jquery.pnotify/jquery.pnotify.d.ts

```diff
@@ -24,25 +29,112 @@ interface PNotifyLabel {
     stick?: string;
 }
 
+
+interface IPNotifyconfirmButton {
```

Comment:
http://www.typescriptlang.org/docs/handbook/writing-declaration-files.html
> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.